index.rst

```diff
@@ -30,6 +30,7 @@ Manage your projects
 
     Delete a project <delete-project>
     Use our backup system <backup-project>
+    Duplicate a project <copy-project>
```

Comment:
Move this up so that all `xxxx a project` entries are listed together. _Duplicate_ should be before _delete_.